addons/web/static/src/core/utils/objects.js

```diff
@@ -22,6 +22,22 @@ export function deepCopy(obj) {
     return JSON.parse(JSON.stringify(obj));
 }
 
+/**
+ * Returns a shallow copy of minuend with every property present in subtrahend
+ * removed.
+ *
+ * @param {Object} minuend object to copy
+ * @param {Object} subtrahend object with the properties to remove
+ * @return {Object}
+ */
+export function omit(minuend, subtrahend) {
```

Comment:
What about an API closer to Omit in Typescript where the second argument is the keys to omit?
```js
omit({ a: 2, b: 2, v: 2 }, ["a", "c"])
```
IMHO it's simpler and makes it very obvious what the function is doing.